WalletWasabi.Tests/UnitTests/BlockchainAnalysis/CoinJoinAnonscoreTests.cs

```diff
@@ -121,7 +121,7 @@ public void MultiDenominationInheritance()
 		[Fact]
 		public void SelfAnonsetSanityCheck()
 		{
-			// If we have multiple same denomination in the same coinjoin, then don't gain anonset on ourselves.
+			// If we have multiple same denomination in the same coinjoin, then our anonset would be total coins/our coins.
```

Comment:
```suggestion
			// If we have multiple same denomination in the same CoinJoin, then our anonset would be total coins/our coins.
```

just pointing out a tiny nit here...